contracts/contracts/validator-registry/avs/MevCommitAVS.sol

```diff
@@ -204,21 +165,31 @@ contract MevCommitAVS is IMevCommitAVS, MevCommitAVSStorage,
     /// @dev Allows a validator to request deregistration from the MevCommitAVS.
     /// @notice For each validator the underlying _requestValidatorDeregistration enforces the sender is either
     /// the podOwner, delegated operator, or the contract owner.
-    function requestValidatorsDeregistration(bytes[] calldata valPubKeys)
-        external whenNotPaused() onlyRegisteredValidators(valPubKeys) onlyPodOwnerOrOperatorOfValidators(valPubKeys) {
+    function requestValidatorsDeregistration(bytes[] calldata valPubKeys) external whenNotPaused() {
         uint256 len = valPubKeys.length;
         for (uint256 i = 0; i < len; ++i) {
```

Comment:
nit to reduce storage reads:
`bytes valPubKey = valPubKeys[i];`
and use that value and same approach later on

Reply:
Fixed in https://github.com/primev/mev-commit/commit/013cc5029a2b0ab567d388084e7bff5c7d443a55